Gopkg.toml

```diff
@@ -76,6 +76,12 @@
   [override.metadata]
     govendor-overriden=true
 
+[[override]]
+  name = "github.com/boombuler/barcode"
```

Comment:
Sheesh, I wonder what's pulling this in...

Reply:
Looks to me to be Vault's client.